tests/clickhouse-test

```diff
@@ -1387,6 +1414,7 @@ if __name__ == '__main__':
     group.add_argument('--shard', action='store_true', default=None, dest='shard', help='Run sharding related tests (required to clickhouse-server listen 127.0.0.2 127.0.0.3)')
     group.add_argument('--no-shard', action='store_false', default=None, dest='shard', help='Do not run shard related tests')
 
+    group.add_argument('--backward-compatibility-check', action='store_true', default=False, help='Ignore all drop queries in tests')
```

Comment:
It's not really clear from help message how it's connected to `backward-compatibility-check`, maybe add a bit more details?

Reply:
Done